tensorflow/c/tf_status.h

```diff
@@ -31,9 +31,13 @@ limitations under the License.
 #endif  // SWIG
 
 // For propagating errors when calling a function.
-#define TF_C_RETURN_IF_ERROR(_status, _value)        \
-  do {                                               \
-    if (TF_GetCode(_status) != TF_OK) return _value; \
+#define TF_C_RETURN_IF_ERROR(_status, _value)           \
+  do {                                                  \
+    if (TF_GetCode(_status) != TF_OK) {                 \
+      printf("Error Code: %d, Error Message: %s\n",     \
+             TF_GetCode(_status), TF_Message(_status)); \
```

Comment:
This should be logging so users can control whether to display it or not

Reply:
It seems that Tensorflow C does not provide a logging function. As we avoid using `core/platform/env/logging`, should we write another logging function ?